sway/workspace.c

```diff
@@ -255,5 +255,18 @@ void layout_log(const swayc_t *c, int depth) {
 		for (i = 0; i < depth; ++i) fputc(' ', stderr);
 		fprintf(stderr,")\n");
 	}
+	if (c->type == C_WORKSPACE) {
+		e = c->floating?c->floating->length:0;
+		for (i = 0; i < depth; ++i) fputc(' ', stderr);
```

Comment:
This was just me extending @taiyu-len's debugging to work on the floating windows. The whole thing is written in that style, and given that you intend to rewrite this, I'm not sure I see a point in adhering to the proper style for now. If you want me to, I'll fix it though.